### ISSUE TITLE
Space Application Supporter can access specific app GET endpoints

### ISSUE THREAD

**Issue by MerricdeLauney** [cloudfoundry/cloud_controller_ng#2209]
## Issue

Allow space application supporter to access specific app endpoints. 

## Context

We are introducing a [new role](https://github.com/cloudfoundry/cfar-proposals/issues/22) and we want to make sure it has the right access.

## Expected result
A space application support should be able access the following endpoints:

- [x] GET /v3/apps/:guid
- [x] GET /v3/apps
- [x] GET /v3/apps/:guid/builds
- [x] GET /v3/apps/:guid/droplets/current
- [x] GET /v3/apps/:guid/relationships/current_droplet
- [x] GET /v3/apps/:guid/permissions

## Acceptance
A space application supporter would see the same info as a space developer assigned to the same space for these and only these endpoints.

## Documentation
When I browse to any of these endpoints on [v3 docs](https://v3-apidocs.cloudfoundry.org/version/3.99.0/index.html) I can see the Space Application Supporter role in the list of permitted roles with an indication that this role is not fully implemented and the permissions will be changing.

**Comment by monamohebbi** [comment]
For `GET /v3/apps/:guid/permissions` we are responding with the basic response:

```
{
  "read_basic_data": true,
  "read_sensitive_data": false
}
```

**Comment by MerricdeLauney** [comment]
@ctlong and I aren't finding the requested documentation changes despite this issue being closed. Was this missed in the PR?

**Comment by sweinstein22** [comment]
Good catch, thanks @MerricdeLauney ! Looks like we did miss it, as it's just docs changes I've added them in [this commit](https://github.com/cloudfoundry/cloud_controller_ng/commit/8afc9c99e3bdd519c37ff3982cc127a6e40fe9ef). and confirmed that the changes load correctly when running the docs locally. We'll do a second round of acceptance just on the docs once the changes get through the pipeline before closing out the issue again.
